src/components/bulkConcepts/component/ActionButtons.jsx

```diff
@@ -63,7 +63,12 @@ export class ActionButtons extends Component {
           type="submit"
           className="btn btn-sm mb-1 actionaButtons"
           id="add-button"
-          onClick={() => this.addConceptButton()}
+          onClick={() => this.addConceptButton(
```

Comment:
Was refactoring part of the task? If it was, I think it should be added to the task description.

Reply:
This was a change related to testing. So I will add it to the description.